server/app/services/languageProcessor/TaskMap.java

```diff
@@ -95,13 +118,23 @@ public JsonNode getTicketAssignee(String issueKey, JsonNode responseBody) {
    */
   public static JsonNode parseToJson(String status, String message) {
```

Comment:
Maybe rename this to `botResponseToJson` and change the parameter to `BotResponse botResponse`. Then the method can only `return Json.toJson(botResponse)`.